components/tables/podcast/LatestEpisodeRow.vue

```diff
@@ -259,22 +265,12 @@ export default {
         this.$eventBus.$emit('pause-item')
       } else {
         this.$store.commit('setPlayerIsStartingPlayback', this.episode.id)
-
-        if (this.localEpisode && this.localLibraryItemId) {
-          console.log('Play local episode', this.localEpisode.id, this.localLibraryItemId)
-
-          this.$eventBus.$emit('play-item', {
-            libraryItemId: this.localLibraryItemId,
-            episodeId: this.localEpisode.id,
-            serverLibraryItemId: this.libraryItemId,
-            serverEpisodeId: this.episode.id
-          })
-        } else {
-          this.$eventBus.$emit('play-item', {
-            libraryItemId: this.libraryItemId,
-            episodeId: this.episode.id
-          })
-        }
+        this.$eventBus.$emit('play-item', {
+          libraryItemId: this.libraryItemId,
+          episodeId: this.episode.id,
+          serverLibraryItemId: this.libraryItemId,
+          serverEpisodeId: this.episode.id
+        })
```

Comment:
Is there a reason for removing the local playback here?

Reply:
I see now. There is a bug in `latest.vue` where localLibraryItemId is always null. I'm looking into this

Reply:
I moved to the `AudioPlayerContainer.vue` to centralize the handling of local files and avoid showing the confirmation dialog for these media states.

Reply:
I fixed this and playing local is possible now

Reply:
For latest podcast episodes we were never passing through the local IDs so downloaded episodes were still direct streaming